public/html/modal.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html class="mode__extension">
+<html class="mode__modal">
```

Comment:
This change is a little misleading, essentially there are now 3 css class modes:

`full-page` (full page view)
`popup` (browser popup)
`modal` (center modal/popup)

Everything associated with the old `centerPopup` has been renamed to `modal` to keep in sync with this naming.